code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1013,6 +1013,22 @@
 	else
 		holder.clear_reagents()
 
+/datum/chemical_reaction/more_bicarodyne
+	name = "Bicarodyne"
+	id = BICARODYNE
+	result = BICARODYNE
+	required_reagents = list(BICARIDINES = 1)
+	required_catalysts = list(BICARODYNE = 1)
+	result_amount = 1
```

Comment:
I think your changelog implies that if you put in 25u bicaridine 25u bicarodyne you end up with 50u bicarodyne. I think it might be more accurate to describe this behavior as "syndicate chems delete their normal counterparts"

Reply:
same thing, no?

Reply:
Oh, you're right, I misread this, yes of course you are just using the bicarodyne as a catalyst not a reagent